tests/conftest.py

```diff
@@ -101,7 +101,8 @@ def factory(interface=constants.CEPHBLOCKPOOL):
             ).get(defaults.CEPHFILESYSTEM_NAME)
             ceph_pool_obj = OCS(**cfs)
         assert ceph_pool_obj, f"Failed to create {interface} pool"
-        instances.append(ceph_pool_obj)
+        if interface == constants.CEPHBLOCKPOOL:
```

Comment:
Just cosmetic proposal, if we want to avoid deleting cephfilesystem, I would maybe use "negative" condition:
```suggestion
        if interface != constants.CEPHFILESYSTEM:
```
(If I understood the code correctly, it should have basically the same effect, but from my POV it will be slightly more readable, WDYT?)

Reply:
ack - fixing now